locale/pt_BR/admin.xml

```diff
@@ -93,8 +93,10 @@
 	<message key="admin.scheduledTask.startTime">Tarefa iniciada.</message>
 	<message key="admin.scheduledTask.stopTime">Tarefa parada.</message>
 	<message key="admin.scheduledTask.noLog">Tarefa não produziu registros (logs).</message>
-	<message key="admin.scheduledTask.openAccessNotification">Notificação de acesso aberto</message>
```

Comment:
Wait what? It got removed in https://github.com/pkp/pkp-lib/commit/13baf96a2f7b173487fce7955d36a0ef67448557.

I'm gonna check to see if anybody add it back.